src/pages/index.js

```diff
@@ -82,87 +59,105 @@ const api = new Api({
     "Content-Type": "application/json",
   },
 });
-function createCard(item) {
-  // create instance of Card class
-  const card = new Card(
-    item,
-    "#card-template",
-    onCardClick,
-    handleCardDeleteFunctionInIndexComponent
-  );
-  //create a card by calling getCardElement method from Card class
-  const cardElement = card.getCardElement();
-  //return the card
-  return cardElement;
-}
-
-api.getUserInfo().then((data) => {
-  const name = data.name;
-  const subtitle = data.about;
-  const url = data.avatar;
-  userInfo.setUserInfo({ name, subtitle });
-  userInfo.setNewAvatar(url);
-});
+const section = new Section(
+  { items: api.getInitialCards, renderer: createCard },
+  constant.cardsList
+);
 
 api
-  .getInitialCards()
-  .then((data) => {
-    // process the result
-    data.forEach((item) => {
-      const cardItem = createCard(item);
-      section.appendItem(cardItem);
+  .getUserInfo()
+  .then((userData) => {
+    userInfo = new UserInfo({
+      userData,
+      selectors: {
+        name: constant.profileTitle,
+        subtitle: constant.profileSubtitle,
+        link: constant.profileAvatar,
+      },
     });
+    userInfo.setUserFields();
   })
   .catch((err) => {
-    console.error(err); // log the error to the console
+    console.log(err);
```

Comment:
Please, delete `api.getUserInfo()` from line 93 because you call it in line 84 inside `Promise.all`. You get all info there